ipyleaflet/leaflet.py

```diff
@@ -329,6 +329,15 @@ class VideoOverlay(RasterLayer):
 
 
 class Velocity(Layer):
+    try:
+        import xarray
+    except ModuleNotFoundError:
+        raise ImportError(
+            'The `xarray` module (required for use of the Velocity layer) is'
+            'not installed. Run "$ pip install xarray" to install the latest'
+            'version.'
```

Comment:
Should we tell how to install? pip is not the only way.

Reply:
I added the instruction for installing with conda as well.

Reply:
I meant, maybe we should not say how to install. Sorry if it was not clear.

Reply:
Oh I see. I do think having the instructions might be nice for someone new to these packages but I could remove them to make the message simpler.

Reply:
I'd rather not have them. @martinRenou what do you think?
(and you don't need dask, netCDF4 or bottleneck in order to install xarray)

Reply:
I agree. Although for beginners it's nice to have an install instruction. But we should not take any guess on what kind of tool they use to install their packages, and telling someone to install something with `pip` when they actually use `conda` might break their environment if they are not careful.